### ISSUE TITLE
GraphQL code is unnecessarily included

### ISSUE THREAD

**Issue by RubenVerborgh** [RubenVerborgh/LDflex-Comunica#19]
Inspecting the generated package with `webpack-bundle-analyzer`, I noticed that GraphQL code was also included because that is required by `@comunica/actor-init-sparql/engine-default.js`. Would should probably have a learner config than that default.

**Comment by rubensworks** [comment]
This probably won't be a quick-fix. We'll have to refactor stuff a bit for this (see https://github.com/comunica/comunica/issues/267)

**Comment by RubenVerborgh** [comment]
> This probably won't be a quick-fix.

We can webpack them out for now.

Also the crypto stuff is very heavy; we can likely do with https://github.com/LinkedDataFragments/Client.js/blob/master/lib/browser/crypto.js

**Comment by rubensworks** [comment]
> Also the crypto stuff is very heavy

This is extensively being used by sparqlee. Since LDflex does not really makes use of filters at the moment, we could probably remove these from the configs as well. And then a more lightweight crypto implementation would be perfect indeed.

**Comment by RubenVerborgh** [comment]
> This is extensively being used by sparqlee.

Also for hashing though.

**Comment by rubensworks** [comment]
> Also for hashing though.

Indeed. Only the distinct/reduced actors do hashing I think. But AFAIK LDflex doesn't use these either, so we can probably remove these from the config as well.
